src/analyzer/protocol/ftp/FTP.cc

```diff
@@ -172,6 +177,17 @@ void FTP_Analyzer::DeliverStream(int length, const u_char* data, bool orig) {
             }
         }
 
+        if ( reply_code == 234 && auth_requested.size() > 0 && auth_requested == "TLS" ) {
```

Comment:
```suggestion
        if ( reply_code == 234 && auth_requested == "TLS" ) {
```

Sorry, I missed that, think the `.size()` is redundant with the strcmp, unless you wanted to save a few cycles here on purpose? Your call.

Reply:
It's analogous to how the GSSAPI check does things. Looks the same as the other code, if nothing else.